static/js/containers/ProfilePage_test.js

```diff
@@ -189,19 +177,20 @@ describe("ProfilePage", function() {
           work_history: [],
         };
         helper.profileGetStub.withArgs(SETTINGS.user.username).returns(Promise.resolve(updatedProfile));
-        return renderComponent(`/profile/${step}`, SUCCESS_ACTIONS).then(([, div]) => {
-          // close all switches
-          if (step === 'personal') {
-            return confirmSaveButtonBehavior(updatedProfile, {div: div}, true);
-          }
-          return confirmSaveButtonBehavior(updatedProfile, {div: div});
+        let actions = step === PERSONAL_STEP ? SUCCESS_ACTIONS.concat(SET_PROGRAM) : SUCCESS_ACTIONS;
```

Comment:
`SUCCESS_ACTIONS.concat(SET_PROGRAM)` is used 4 times. can we just define a new constant for it? `SUCCESS_AND_SET_PROGRAM_ACTIONS` or something? that name sucks. if you think of a better one, go for it. otherwise you can ignore this